dist/gcc-compatible/lib_intrinsics.h

```diff
@@ -18,18 +18,32 @@
 
 #include "Hacl_IntTypes_Intrinsics.h"
 
-#define Lib_IntTypes_Intrinsics_add_carry_u32(x1, x2, x3, x4) \
```

Comment:
the second one is indeed redundant because I think ARMv8 mandates NEON (I think)

Reply:
Looking around it does seem that aarch64 implies NEON. I think then the only strangeness might come in case an ARM8 processor is running in aarch32 mode, but I think in this case it would register as ARM7 to the configure script, right? I'm not sure how this works though.

Reply:
I would think so, yes. Or, it's a cross-compilation scenario.